Assets/Scripts/Windows/GenericWinow.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+public class GenericWinow : MonoBehaviour
```

Comment:
Nitpick: typo for "GenericWinow" -> "GenericWindow"

Reply:
It builds though! :P

Reply:
I still suggest to fix spelling errors to make it look professional.